main.cpp

```diff
@@ -171,7 +175,14 @@ std::vector<Element> Storage::GetLogs(time_t start, time_t finish)
     {
         if (elem.GetTime() >= start && elem.GetTime() <= finish)
         {
-            dates.push_back(elem);
+            if(dates.empty()||dates[0].GetTime()<elem.GetTime())
+            {
+                dates.push_back(elem);
+            }
+            else
+            {
+                dates.insert(dates.begin(), elem);
```

Comment:
this line will never execute? since dates[0] will aways have an element that was inserted before next ones in the array _new?

Reply:
yes. It's a mistake